src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        storageFruits.put(transaction.getFruit(), storageFruits
+                .getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
+    }
+}
```

Comment:
According to the checklist, the classes should follow SOLID principles. The 'ReturnOperation' class directly modifies the 'storageFruits' which seems to be a global state. This violates the single responsibility principle as the class is responsible for both the operation logic and interacting with the storage. Consider refactoring this by injecting a storage service into the operation handler that can be used to modify the storage state.